modules/ROOT/pages/maven-reference.adoc

```diff
@@ -77,6 +77,73 @@ When you create a new Mule project in Anypoint Studio, the required Mule reposit
 </pluginRepository>
 ----
 
+== MuleSoft Enterprise Repository
+
+To add the MuleSoft Enterprise repository to your project, you need to meet the following prerequisites:
+
+* Own an Enterprise license
++
+If you need instructions to acquire or install an Enterprise License, see xref:installing-an-enterprise-license.adoc[Installing an Enterprise License]
+* Have valid credentials to access the https://repository.mulesoft.org/nexus-ee/content/repositories/releases-ee/[MuleSoft Enterprise repository]
++
+If you are a MuleSoft customer and do not have access to the repository, contact Support and request Nexus enterprise credentials.
+
+=== Configure MuleSoft Enterprise Repository
+
+To configure Maven to access the MuleSoft Enterprise Repository:
+
+. Inside `<USER_HOME>/.m2/settings.xml`, open the `settings.xml` config file on all workstations that require access.
+** If the file does not exist, create it.
```

Comment:
Why is this a subbullet?
And which file is "the file"?
Should this be...:
 
. Navigate to <blah>, or create it, if it does not exist.
. Open `settings...` for every workstation....
. Add the following syntax to the [each?] `servers` section: